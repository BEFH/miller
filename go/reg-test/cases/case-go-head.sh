run_mlr --opprint head -n 1 $indir/s.dkvp
run_mlr --opprint head -n 1 -g a $indir/s.dkvp
run_mlr --opprint head -n 1 -g a,b $indir/s.dkvp
