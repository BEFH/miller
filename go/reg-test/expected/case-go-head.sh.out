
mlr --opprint head -n 1 ./reg-test/input/s.dkvp
a   b   i x                  y
pan pan 1 0.3467901443380824 0.7268028627434533

mlr --opprint head -n 1 -g a ./reg-test/input/s.dkvp
a   b   i x                   y
pan pan 1 0.3467901443380824  0.7268028627434533
eks pan 2 0.7586799647899636  0.5221511083334797
wye wye 3 0.20460330576630303 0.33831852551664776

mlr --opprint head -n 1 -g a,b ./reg-test/input/s.dkvp
a   b   i x                   y
pan pan 1 0.3467901443380824  0.7268028627434533
eks pan 2 0.7586799647899636  0.5221511083334797
wye wye 3 0.20460330576630303 0.33831852551664776
eks wye 4 0.38139939387114097 0.13418874328430463

